src/Controls/tests/Xaml.UnitTests/XmlnsCollision.xaml.cs

```diff
@@ -0,0 +1,66 @@
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Build.Tasks;
+using Microsoft.Maui.Controls.Core.UnitTests;
+using NUnit.Framework;
+
+[assembly: XmlnsDefinition("http://companyone.com/schemas/toolkit", "CompanyOne.Controls")]
+[assembly: XmlnsPrefix("c1", "http://companyone.com/schemas/toolkit")]
```

Comment:
Is it expected to use these two together? Would it be possible to allow the developer to completely skip declaring the URL and just do something like `[assembly: XmlnsPrefix("c1", "CompanyOne.Controls")]`?